src/content/documentation/develop/debugging.md

```diff
@@ -383,4 +383,10 @@ For more details, see [Testing persistent and restart features](/documentation/d
 
 <!-- END: Single Column Body Module -->
 
-
+[https://developer.mozilla.org/docs/Tools/Page_Inspector]:https://developer.mozilla.org/docs/Tools/Page_Inspector
```

Comment:
Would be nice to not use the link as the ref for all of these. See other examples and use text-only references where possible.